indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -0,0 +1,148 @@
+{% macro render_registration_info(registration) %}
+    {% for section in registration.registration_form.sections -%}
+        <tr>
+            <td class="regform-done-title">{{ section.title }}</td>
+        </tr>
+        {% for field in section.children if field.is_field and field.id in registration.data_by_field -%}
+            <tr>
+                <td class="regform-done-caption">{{ field.title }}</td>
+                <td class="regform-done-data">
+                    {{ render_data(registration, field) }}
+                </td>
+            </tr>
+        {%- endfor %}
+    {%- endfor %}
+{% endmacro %}
+
+{% macro render_data(registration, field) %}
+    {% set friendly_data = registration.data_by_field[field.id].friendly_data %}
+    {% if field.input_type == 'accommodation' %}
```

Comment:
`and value` or it will fail if someone didn't select anything in an accommodation field. Or do we assume that we always have data? Just noticed that the field is always required